space-template.yaml

```diff
@@ -0,0 +1,172 @@
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
+# Although we have the field types above we cannot always know, just by looking
+# at the field type, how a particular field type shall be rendered. That's why
+# we have these layout options.
+#ui_layouts:
+#  input
+
+# A map of common strings that can be used to denote states of a work item
+common_work_item_states: &common_work_item_states
+  &state_new new:
+      en_US: "new"
+      de_DE: "neu"
+  &state_open open:
+      en_US: "open"
+      de_DE: "offen"
+  &state_closed closed:
+      en_US: "closed"
+      de_DE: "geschlossen"
+  &state_resolved resolved:
+      en_US: "resolved"
+      de_DE: "gelöst"
+  
+# A map of common fields in a work item type
+work_item_fields_bag:
+  field_title: &field_title
+    label:
+      en_US: "Title"
+      de_DE: "Titel"
+    description:
+      en_US: "A small text title that is displayed in lists and alike"
+      de_DE: "Ein kurzer Text, der in Listen angezeigt wird"
+    type: *string
+    required: true
+  field_description: &field_description
+    label:
+      en_US: "Description"
+      de_DE: "Beschreibung"
+    description:
+      en_US: "A longer text that is only displayed in places with more space"
+      de_DE: "Ein längerer Text, der an Stellen mit mehr Platz angezeigt wird"
+    type: *string
+    required: false
+  field_created_at: &field_created_at
+    label:
+      en_US: "Created at"
+      de_DE: "Erstellt am"
+    description:
+      en_US: "The timestamp of when the work item was created"
+      de_DE: "Der Zeitstempel, zu dem dieses Work Item erstellt wurde"
+    type: *timestamp
+    required: false
+  field_updated_at: &field_updated_at
+    label:
+      en_US: "Updated at"
+      de_DE: "Aktualisiert am"
+    description:
+      en_US: "The timestamp of when the work item was last updated"
+      de_DE: "Der Zeitstempel, zu dem dieses Work Item zuletzt aktualisiert wurde"
+    type: *timestamp
+    required: false
+  field_state: &field_state
+    label:
+        en_US: "State"
+        de_DE: "Status"
+    description:
+        en_US: "A work item can have different states (e.g. new, open, or closed)."
+        de_DE: "Ein Fehler kann mehrere Zustände besitzen."
+    type: *list
+    element_type: *translatable_string
+    default_value: *state_new
+    values: *common_work_item_states
+    required: false
+
+# If no other fields are required for your PR you can use this handy default
+common_work_item_fields: &common_work_item_fields
+  system_title: *field_title
+  system_description: *field_description
+  system_created_at: *field_created_at
+  system_updated_at: *field_updated_at
+  system_state: *field_state
+
+# This map defines all the work item types that exist in a space. We re-use some
+# of the above states and lists in order share code between work item types.
+work_item_types:
```

Comment:
@kwk My comment was specific to a process type - these seem to be commingled, not strictly Agile Backlog types or Scenario Based Development process types - but both intermingled.